docker-compose.yml

```diff
@@ -0,0 +1,20 @@
+services:
+  ###
+  # Main php container
+  #
+  # This container runs the php-fpm process and serves WordPress.
+  #
+  # Requires the db container to be running.
+  #
+  # Contains:
+  #  - php-fpm, xdebug, composer
+  #  - WordPress installation (in volume web-data)
+  #  - Various tools: zsh, vim
+  #
+  # See https://github.com/myparcelnl/docker-images/pkgs/container/wordpress
+  ##
+  wordpress:
```

Comment:
waarom in top level directory?

Reply:
Dan heb je de containers overzichtelijk bij elkaar als je ze wilt opspinnen om te testen `doco up wordpress` en wordt de map met code van de container zelf niet vervuild met een docker-compose file die je verder niet gebruikt.

Reply:
Dat is niet waar deze repo voor bedoeld is (of was), maar voor nu vind ik het prima, kunnen we later altijd nog naar kijken